inference/python/infbench/cutlass.py

```diff
@@ -93,33 +191,38 @@ class cutlassSgemmLoader(dataset.loader):
     checkAvailable = True
 
     def __init__(self, dataDir):
-        self.M = 10000
-        self.N = 8000
+        self.M = 1000
```

Comment:
These constants show up in both the loader and the model. We should probably put those into global constants so we only have to change one place (e.g. make globals for M,N,K and use those to assign to the loader and model)